src/test/regress/expected/background_rebalance_parallel.out

```diff
@@ -0,0 +1,176 @@
+  17777 | 1460001 |    1460000
+  17777 | 1460002 |    1460001
+  17777 | 1460003 |    1460002
+  17777 | 1460004 |    1460003
+  17777 | 1460005 |    1460004
+  17777 | 1460006 |    1460005
+  17777 | 1460007 |    1460006
+(7 rows)
+
+/* Check that if there is a reference table that needs to be synched to a node, the first move scheduled in a colocation group takes a dependency on the
+   replicate_reference_tables task */
+SELECT 1 FROM citus_drain_node('localhost',:worker_2_port);
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+NOTICE:  Moving shard xxxxx from localhost:xxxxx to localhost:xxxxx ...
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+SELECT 1 FROM citus_disable_node('localhost', :worker_2_port, synchronous:=true);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+CREATE TABLE ref_table(a int PRIMARY KEY);
+SELECT create_reference_table('ref_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+/* Activate a node so that we can rebalance */
+SELECT 1 FROM citus_activate_node('localhost', :worker_2_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+SELECT citus_set_node_property('localhost', :worker_2_port, 'shouldhaveshards', true);
+ citus_set_node_property
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT * FROM citus_rebalance_start();
+NOTICE:  Scheduled 8 moves as job xxx
+DETAIL:  Rebalance scheduled as background job
+HINT:  To monitor progress, run: SELECT * FROM citus_rebalance_status();
+ citus_rebalance_start
+---------------------------------------------------------------------
+                 17778
+(1 row)
+
+SELECT citus_rebalance_wait();
+ citus_rebalance_wait
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT * FROM pg_dist_background_task_depend WHERE job_id = 17778 ORDER BY task_id ASC;
+ job_id | task_id | depends_on
+---------------------------------------------------------------------
+  17778 | 1460009 |    1460008
+  17778 | 1460010 |    1460009
+  17778 | 1460011 |    1460010
+  17778 | 1460012 |    1460011
+  17778 | 1460013 |    1460012
+  17778 | 1460014 |    1460013
+  17778 | 1460015 |    1460014
+  17778 | 1460016 |    1460015
+(8 rows)
+
+DROP SCHEMA background_rebalance_parallel CASCADE;
```

Comment:
Only showing warnings removes the logs at the end.
```suggestion
SET client_min_messages TO WARNING;
DROP SCHEMA background_rebalance_parallel CASCADE;
```

Usually I create new tests file with `src/test/regress/bin/create_test.py`, because that takes care of setting up all this boilerplate.